server/worker/jira.go

```diff
@@ -1,31 +1,152 @@
 package worker
 
 import (
+	"bytes"
 	"context"
 	"encoding/json"
+	"html/template"
 
+	jira "github.com/andygrunwald/go-jira"
+	"github.com/fleetdm/fleet/v4/server/contexts/ctxerr"
 	"github.com/fleetdm/fleet/v4/server/fleet"
 	kitlog "github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
 )
 
-type Jira struct {
-	ds  fleet.Datastore
-	log kitlog.Logger
-	// TODO: add jira client
+const (
+	// JiraName is the name of the job as registered in the worker.
+	JiraName = "jira"
```

Comment:
Does this need to be exported?

Reply:
At some point it was, but looks like it isn't anymore, will unexport.